lib/resources/r4/related_person_patch.yaml

```diff
@@ -159,3 +159,32 @@ operations:
         <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `family` is intended to be replaced.</li>
         <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
       </ul>
+
+  - name: test-telecom-id
+    path: /telecom/{index}/id
+    operation: test
+    type: string
+    url: https://hl7.org/fhir/R4/element-definitions.html#Element.id
+    description: Test the value of the `ContactPoint.id` of the telecom at the given <code>{index}</code> in the list of related person telecoms.
+    example: |
+      {
+        "op": "test",
+        "path": "/telecom/0/id",
+        "value": "CI-PH-2807950-0"
+      }
+
+  - name: remove-telecom
+    path: /telecom/{index}
+    operation: remove
+    type: N/A
+    url: https://hl7.org/fhir/R4/patient-definitions.html#related person.telecom
```

Comment:
I think the url will not be working. First of all patient-definitions will take to the patient page, which is wrong and then #related person will not point to any specific field.
So, it should ideally be:  https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.telecom

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/commit/150edb0c78f14b75bf4da4f1412e24b65b90c50a

Reply:
Now when I recheck I think the link I provided was also incorrect, sorry for that.
It should be: https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.telecom

Please check the link once to see if it is taking you to the page, before you push it.

Reply:
Updated the url: https://github.com/cerner/fhir.cerner.com/commit/8c298287e4acd1221bb0bdd92719a25868a8496b